build.gradle

```diff
@@ -73,11 +73,12 @@ dependencies {
     compile 'org.aspectj:aspectjtools:1.7.3'
     compile 'org.aspectj:aspectjweaver:1.7.3'
     compile 'org.aspectj:aspectjrt:1.7.3'
-    compile 'com.google.guava:guava:15.0'
+    compile 'com.google.guava:guava:18.0'
```

Comment:
this is probably not gonna fly since we have guava 11.0.2 on dataproc machines...